pkg/azurefile/azurefile.go

```diff
@@ -108,28 +115,51 @@ func (d *Driver) Run(endpoint string) {
 	s.Wait()
 }
 
-func (d *Driver) checkFileShareExists(accountName, resourceGroup, name string) (bool, error) {
+func (d *Driver) checkFileShareExists(accountName, resourceGroup, name string, accountKey *string) (bool, error) {
 	// find the access key with this account
-	accountKey, err := d.cloud.GetStorageAccesskey(accountName, resourceGroup)
+	key, err := d.cloud.GetStorageAccesskey(accountName, resourceGroup)
 	if err != nil {
 		// Because the account does not exist in the resource group, the file share could not exist.
 		// In the next step, we will create a temp account to create file share.
 		// So here we return nil, and warn the user.
 		klog.Warningf("could not get storage key for storage account %s: %v", accountName, err)
```

Comment:
pls remove this comment, could not find the storage account key is a common scenario when user does not input `accountName`